<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="Grahps_list">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_values">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Мои графики:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_graph_name">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>График</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_delete_graph"/>
    <addaction name="action_create_graph"/>
    <addaction name="action_edit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_delete_graph">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_create_graph">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Построить </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
